### ISSUE TITLE
functions to calculate a vectors magnitude and angle

### ISSUE THREAD

**Issue by 4lbert** [japaric/fpa#2]

**Comment by japaric** [comment]
That may be a convergence bug. `3.polar(4)` does return `5`.

**Comment by 4lbert** [comment]
Yeah, seems like this happens when both numbers are too close to each other. Thanks, guess I'll just ignore when this causes bugs, shouldn't be too often after all.
